netty/src/main/java/io/grpc/netty/NettyClientTransport.java

```diff
@@ -75,6 +79,11 @@
   private final int flowControlWindow;
   private final int maxMessageSize;
   private final int maxHeaderListSize;
+  private ScheduledExecutorService scheduler;
+  private KeepAliveManager keepAliveManager;
+  private boolean enableKeepAlive;
+  private long keepAliveDelayNanos;
+  private long keepAliveTimeoutNanos;
```

Comment:
I think I'll refactor this to be builder. I have next PR lined up which will add 2 more attrs here